k2/python/tests/ragged_shape_test.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+#
+# Copyright (c)  2020  Mobvoi Inc.        (authors: Liyong Guo)
+#
+# See ../../../LICENSE for clarification regarding multiple authors
+
+# To run this single test, use
+#
+#  ctest --verbose -R  ragged_shape_test_py
+
+import unittest
+
+import k2
+import torch
+
+
+class TestIndex(unittest.TestCase):
```

Comment:
Please change `TestIndex` to `TestRaggedShape`.

Reply:
fixed.